src/components/timecard/simple-time-period/SimpleTimePeriod.spec.jsx

```diff
@@ -38,10 +38,14 @@ jest.mock('../../../api/services/timecardService', () => ({
   deleteTimeEntry: jest.fn().mockResolvedValue({ status: 200 }),
 }));
 
-describe('ScheduledRestDay', () => {
+describe('SimpleTimePeriod', () => {
   it('should display the correct time period type', () => {
     renderWithTimecardContext(
-      <ScheduledRestDay timeEntry={newTimeEntry} timeEntriesIndex={0} />
+      <SimpleTimePeriod
+        timeEntry={newTimeEntry}
+        timeEntriesIndex={0}
+        timePeriodTitle="Scheduled rest day"
```

Comment:
nit: I think this title param could have been extracted out to the top level as its used all across this test suite. 
Not necessary to change though